Integrations/integration-SplunkPy.yml

```diff
@@ -413,10 +424,12 @@ script:
         response_info = updateNotableEvents( sessionKey=sessionKey, baseurl=baseurl, comment=demisto.get(demisto.args(), 'comment'), status=status,
             urgency=demisto.get(demisto.args(), 'urgency'),
             owner=demisto.get(demisto.args(), 'owner'), eventIDs=eventIDs)
+        if 'success' not in response_info:
+            return_error("api error, field success is missing. api response: " + str(response_info['messages']))
```

Comment:
I think the error message should be the same as the one below: `Could not update notable events...`